web/client-api/src/main/java/io/deephaven/web/client/api/tree/JsTreeTable.java

```diff
@@ -1036,6 +1132,32 @@ public JsArray<FilterCondition> applyFilter(FilterCondition[] filter) {
         return getFilter();
     }
 
+    /**
+     * Adding new columns to the table based on other columns using updateView() mechanics.
+     *
+     * @param customColumns
+     * @return {@link CustomColumn} array
+     */
+    @JsMethod
+    @SuppressWarnings("unusable-by-js")
```

Comment:
this suppression shouldn't be here, and is probably telling us that something is wrong in our types.

Reply:
Somehow this is back, I thought it was removed before?